ansible/deploy-proteus-events.yml

```diff
@@ -0,0 +1,27 @@
+- hosts: events.proteus.test.ooni.io
+  vars:
+    proteus_events_https_port: 443
+    letsencrypt_nginx: yes
+    letsencrypt_domains: "{{ inventory_hostname }}"
+    proteus_database_url: "postgres://proteus:{{ proteus_database_password_testing }}@db-1.proteus.test.ooni.io:5432/proteus?sslmode=require"
```

Comment:
It's better to protect against a purely passive attack and requiring an active attack to leak the password.
Also it's probably not a big deal since we are communicating inside of our network.